rpcserver.go

```diff
@@ -7073,7 +7073,7 @@ func (r *rpcServer) FundChannel(ctx context.Context,
 	}
 
 	assetID, groupKey, err := parseAssetSpecifier(
-		req.GetAssetId(), "", nil, "",
+		req.GetAssetId(), "", req.GetGroupKey(), "",
```

Comment:
Re the other comment, this is where we can implement the either or behavior: 
  * We only allow asset ID or group key to be set
  * We make a specifier that only has one half populated 

With that change, I don't think we'd need that new bool and behavior change in the db?

Reply:
Hmm, yeah, maybe we can introduce a sub type `ExclusiveSpecifier` that embeds the original identifier (just so we don't need to change things everywhere)... Will think about it.